### DISCUSSION TITLE
SB_BUILDER-WEBPACK5_0003 (WebpackCompilationError): There were problems when compiling your code with Webpack.

### DISCUSSION THREAD

**Discussion by ZaheerNiazi007** [storybookjs/storybook#28911] [Help]
### Summary

I am getting this error since 2 to 3 days is there any solution for this . thank you in advance :) ![image](https://github.com/user-attachments/assets/fabc13aa-b4e7-476a-9856-0a002dfb6ce4)

**Comment by ZaheerNiazi007** [comment]
Storybook Environment Info:

  System:
    OS: Windows 11 10.0.22631
    CPU: (8) x64 Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz
  Binaries:
    Node: 20.8.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.22 - C:\Program Files\nodejs\yarn.CMD <----- active
    npm: 10.8.2 - E:\app-design-system\node_modules\.bin\npm.CMD
  Browsers:
    Edge: Chromium (127.0.2651.86)
  npmPackages:
    @storybook/addon-essentials: ^8.2.9 => 8.2.9
    @storybook/addon-interactions: ^8.2.9 => 8.2.9
    @storybook/addon-links: ^8.2.9 => 8.2.9
    @storybook/addon-onboarding: ^8.2.9 => 8.2.9
    @storybook/blocks: ^8.2.9 => 8.2.9
    @storybook/preset-create-react-app: ^8.2.9 => 8.2.9
    @storybook/react: ^8.2.9 => 8.2.9
    @storybook/react-webpack5: ^8.2.9 => 8.2.9
    eslint-plugin-storybook: ^0.8.0 => 0.8.0
  npmGlobalPackages:
    @storybook/cli: 8.2.9

these are my information

**Comment by Marklb** [comment]
I don't know if there is enough information to really know what is happening.

My first suggestion, when dealing with a Webpack error, is usually to delete `node_modules/` and `package-lock.json`/`yarn.lock`. You could also check your lock file to see if there are any webpack version conflicts, such as a webpack 4 dependency getting used with webpack 5.

If you could reproduce it in a project that you can share that would be best to get an answer. Ideally by configuring a sandbox the same way at https://storybook.new
